resources/etc/cas/config/log4j2.xml.tpl

```diff
@@ -104,6 +116,15 @@
         <AsyncLogger name="io.netty.handler.logging.LoggingHandler" level="${sys:ces.log.level}" includeLocation="true" additivity="false">
             <AppenderRef ref="loggingHandlerPasswordRewritePolicy"/>
         </AsyncLogger>
+        <AsyncLogger name="org.apereo.cas.web" level="${sys:ces.log.level}" includeLocation="true" additivity="false">
```

Comment:
One question, can the loggers below be removed? Since they are in the same package?

Reply:
defaultDelegatedClientIdentityProviderConfigurationProducerRewritePolicy uses a different password matching string, so we cannot remove it